state/protocol/badger/mutator_test.go

```diff
@@ -631,7 +631,7 @@ func TestExtendBlockNotConnected(t *testing.T) {
 		require.NoError(t, err)
 
 		// create a fork at view/height 1 and try to connect it to root
-		extend.Header.Timestamp = extend.Header.Timestamp.Add(time.Second)
+		extend.Header.Timestamp += uint64(time.Second / time.Millisecond)
```

Comment:
functionally the same, but I feel the following is more explicit:

```suggestion
		extend.Header.Timestamp += 1000 // shift time stamp forward by 1 second = 1000ms
```